src/App.tsx

```diff
@@ -1,24 +1,34 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import "./App.css";
+import Image from "./Components/Image";
+import Info from "./Components/Info";
+import UploadFile from "./Components/UploadFile";
+import { useUploadFile } from "./Hooks/useUploadFile";
+import { useMouse } from "./Hooks/useMouse";
 
 function App() {
+  const { data, areas, url, setUrl, setWH, add, deleteArea } = useUploadFile();
+  const { ref, isPress } = useMouse();
+
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.tsx</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div className="App" ref={ref}>
```

Comment:
I used react starter to build this app and that have default css setting.
I thought that use Styled Components is more conform to React Components.
But I still leave `App.css` for global css setting in case of need.